shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/domain/model/Question.kt

```diff
@@ -21,43 +20,87 @@ data class Question(
     val coverImageUrl: String,
     val answers: List<Answer>
 ) {
-    private val sortedAnswers: List<Answer>
+    val sortedAnswers: List<Answer>
         get() = answers.sortedBy { it.displayOrder }
-
-    val answerTexts: List<String>
-        get() = sortedAnswers.map { it.text.orEmpty() }
-
-    val answerPlaceholders: List<String>
-        get() = sortedAnswers.map { it.inputMaskPlaceholder.orEmpty() }
 }
 
 sealed class QuestionDisplayType {
 
-    object Intro : QuestionDisplayType()
-    object Star : QuestionDisplayType()
-    object Heart : QuestionDisplayType()
-    object Smiley : QuestionDisplayType()
-    data class Choice(val answers: List<String>) : QuestionDisplayType()
-    object Nps : QuestionDisplayType()
-    data class Textarea(val placeholder: String) : QuestionDisplayType()
-    data class Textfield(val placeholders: List<String>) : QuestionDisplayType()
-    data class Dropdown(val answers: List<String>) : QuestionDisplayType()
-    object Outro : QuestionDisplayType()
-    object Unsupported : QuestionDisplayType()
+    abstract var answers: List<Answerable>
```

Comment:
It seems we don't change this value, could we change to `val answers`

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/123/commits/8d54f7c935b2f175735e0975538c2350cfc493e3